picard/formats/id3.py

```diff
@@ -595,6 +596,36 @@ def _info(self, metadata, file):
         metadata['~format'] = self.NAME
 
 
+class DSFFile(ID3File):
```

Comment:
We can over-ride the supports_tag method in the class to exclude the unsupported tags so that they don't appear in the UI.